syntax = "proto3";

package meterusage;

service MeterUsageService {
  rpc GetMeterUsage (MeterUsageRequest) returns (MeterUsageResponse) {}
}

message MeterUsageRequest {}

message MeterUsageRecord {
  string time = 1;
  double meterusage = 2;
}

message MeterUsageResponse {
  repeated MeterUsageRecord records = 1;
}